src/ExtInterface/ReefMod/Domain.jl

```diff
@@ -1,5 +1,6 @@
 using NamedDims, AxisKeys, CSV
-using CSV, DataFrames, Statistics, Distributions
+using DataFrames, Statistics, Distributions
+using SparseArrays
```

Comment:
I remember you said you were not going to use SparseArrays anymore. Should this line be removed then?

Reply:
Thanks for the catch